samples/src/main/java/ecom/EcomAsyncJavaClient.java

```diff
@@ -67,4 +77,45 @@ public static void main(String[] args) {
         ecomAsyncJavaClient.getPayments();
         ecomAsyncJavaClient.sendPaymentAsync(amountInPence);
     }
+
+    public static class SendPaymentCommand implements PaymentCommand {
+
+        private static final EcomPaymentInput input = new EcomPaymentInput();
+
+        public SendPaymentCommand() {
+        }
+
+        @Override
+        public void execute() {
+            asyncClient.sendPaymentAsync(input.getAmount(), input.getCurrency(), input.getPaymentReference(), input.getOrderId());
```

Comment:
sendPaymentAsync returns a `CompletableFuture` but this methods returns `void`

Reply:
The returned value was intentionally ignored here.